src/components/platformContent.tsx

```diff
@@ -98,7 +104,7 @@ export function PlatformContent({includePath, platform, children}: Props) {
           <div className="dropdown mr-2 mb-1">
             <button
               className="btn btn-sm btn-secondary dropdown-toggle"
-              onClick={() => setDropdown(!dropdown)}
+              onClick={() => setDropdown(prevValue => !prevValue)}
```

Comment:
What is the dropdown for PlatformContent?

Reply:
Not sure lol